lib/ci/ci_result_parser.js

```diff
@@ -880,6 +919,8 @@ class NormalBuild extends Job {
       return this.failures;
     }
 
+    if (!runs) return;
```

Comment:
This is returning `undefined` (effectively), so now the caller has to be able to deal with that, maybe better to return `[]`? Functions that return an array-or-undefined require callers to check. Returning an array always simplifies things, and allows only callers that _really_ care to check if `retval.length === 0`. If the function really needs to distinguish error from zero-length data, that'd be different, but that is probably not the case here.